_includes/guides.html

```diff
@@ -22,7 +22,7 @@ <h5 class="grid_content grid_content_dark">Overview</h5>
                         <a href="https://rackhd.readthedocs.io/en/latest/tutorials/index.html" class="list-group-item list-group-item-fixed-border">
                              <h5 class="grid_content grid_content_dark">Tutorials</h5>
                         </a>
-                        <a href="https://rackhd.atlassian.net/wiki/display/RAC1/RackHD+Installation+Guide+-1.0.0" class="list-group-item list-group-item-fixed-border">
+                        <a href="https://rackhd.atlassian.net/wiki/display/RAC1/RackHD+Release+Installation+Guide" class="list-group-item list-group-item-fixed-border">
```

Comment:
@anhou  , The "Get RackHD" button now changes to "Release Page" as you suggested.
BUt the "Installation Guide" link in portal still points to the detailed install confluence page.

please sign off.